tools/cockpit.spec

```diff
@@ -124,6 +124,7 @@ if [ -n "%{patches}" ]; then
 	git add -f . && git commit -a -q -m "Base" && git tag -a initial --message="initial"
 	git am --whitespace=nowarn %{patches}
 	touch -r $(git diff --name-only initial..HEAD) .git
+	touch configure.ac aclocal.m4 configure Makefile.am Makefile.in
```

Comment:
aclocal.m4 apparently gets [excluded from patches](https://github.com/cockpit-project/cockpituous/blob/master/release/release-source#L166) for some reason; but even if that's not it, touching that seems harmless to me. However, touching Makefile.in isn't -- a patch that only changes Makefile.am would then just silently be ignored, instead of being flagged broken for forgetting to also change Makefile.in. Would it be possible to add a comment to Makefile.am in affected patches to force rebuilding the necessary Makefile.in bits too? This would be a bit more manual work, but at least err on the safe side.